src/sass/base/_base.scss

```diff
@@ -9,6 +9,18 @@
 html {
   box-sizing: border-box;
   font-size: 62.5%; // 10px / 16px = 62.5% -> 1rem = 10px
+
+  @include respond(large-screen) {
+    font-size: 75%; // 1rem = 12px
+  }
+
+  @include respond(tab-land) {
+    font-size: 56.25%; // 1rem = 9px
+  }
+
+  @include respond(tab-port) {
+    font-size: 50%; // 1rem = 8px;
```

Comment:
8 and 9px are very small font sizes. Why do you need such sizes?

https://www.boia.org/blog/accessibility-tips-let-users-control-font-size

Reply:
These are not the font sizes, these are the values equal to 1rem.
On a normal desktop, 1rem = 10px; on tab-land, 1rem=9px; on tab-port and phone, 1rem = 8px.

By decreasing the value of 1rem, I can handle responsiveness more dynamically on different devices as all the elements using the unit rem will shrink with the device screen.